### PR TITLE
Broadinstitute -> DataBiosphere changes

### PR DESCRIPTION
**Pull request by rushtong**
Addresses https://broadinstitute.atlassian.net/browse/GAWB-3024
Updates for the migration from broadinstitute -> DataBiosphere github/travis orgs.

See also: 
* https://github.com/DataBiosphere/consent-ui/pull/163
* https://github.com/DataBiosphere/consent/pull/249

---

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [ ] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Consent Ontology Services
 
-[![Build Status](https://travis-ci.com/broadinstitute/consent-ontology.svg?token=3ve6QNemvC5zpJzsoKzf&branch=develop)](https://travis-ci.com/broadinstitute/consent-ontology)
+[![Build Status](https://travis-ci.org/DataBiosphere/consent-ontology.svg?token=3ve6QNemvC5zpJzsoKzf&branch=develop)](https://travis-ci.org/DataBiosphere/consent-ontology)
```
Comment (davidangb) [inline README.md]: remove the token, I think. If I look at https://github.com/DataBiosphere/consent-ontology/tree/GAWB-3024 I can see the badge is broken.

Comment (davidangb) [inline README.md]: also maybe add the coverage badge? something like:
```
[![Coverage Status](https://coveralls.io/repos/DataBiosphere/consent-ontology/badge.svg?branch=develop)](https://coveralls.io/r/DataBiosphere/consent-ontology?branch=develop)
```

**Review by davidangb — CHANGES_REQUESTED** [review]

**Review by aednichols — COMMENTED** [review]
Looks fine pending David's comments; will thumb then. Travis is verifiably back.

File: README.md (hunk 1 above)
Comment (rushtong) [reply README.md]: Similar to the comment in consent, we can't link there without a new coveralls account being configured for `DataBiosphere`. Currently looking at other ways to accomplish this same task.

**Comment by davidangb** [comment]
Gotcha on the coveralls badge. I still think the Travis badge needs the token removed, though.

**Pull request merged**